cmd/suggestion/v1alpha2/hyperband/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:alpine AS build-env
+# The GOPATH in the image is /go.
```

Comment:
we had `cmd/suggestion/hyperband/v1alpha1` already. can you move it in `cmd/suggestion/hyperband/v1alpha2`? do dose others